WindowsInstaller/config_template.txt

```diff
@@ -132,13 +132,13 @@ audio_buffer_size = 2048
 -------------------------------------------------------------------------------
 -- If you want to listen to non-Theme-Hospital music, then follow these steps:
 --  1) Find updated versions of the original tracks (for example the remixes by ZR
---     from http://www.a-base.dds.nl/temp/ThemeHospital_ZRRemix.zip ) or any
---     other music you want to listen to.
+--     from https://drive.google.com/file/d/1HuA-mCs87fl7tlfJ4btgeOZVfX5k5Du_/view )
```

Comment:
As we state this as available, we're a distributor. There is a proper license attached to this data?

Also, if we distribute it, why not put it at GH itself, this url doesn't give much confidence imho.

Reply:
I asked the creator on his [youtube](https://www.youtube.com/user/zenithalravage) channel, on a video now private. He gave this link. We could ask for permission to host it ourselves?

Reply:
At least in the US and Germany we would not be infringing by linking to copyrighted material. That was the aspect of SOPA that had the tech companies so up in arms all those years ago.

Reply:
That said I'm leaning against calling it out there. The wiki or the discussion group seems like more appropriate forums.